i18n/en.json

```diff
@@ -2073,7 +2073,7 @@
   "api.channel.post_convert_channel_to_private.updated_from": "{username} converted the channel from public to private",
   "api.channel.post_update_channel_displayname_message_and_forget.updated_from": "{username} updated the channel display name from: {old} to: {new}",
   "api.channel.post_update_channel_header_message_and_forget.removed": "{username} removed the channel header (was: {old})",
-  "api.channel.post_update_channel_header_message_and_forget.updated_from": "{username} updated the channel header from: {old} to: {new}",
+  "api.channel.post_update_channel_header_message_and_forget.updated_from": "{username} updated the channel header <br></br><strong>from:</strong> {old} <br></br><strong>to:</strong> {new}",
```

Comment:
@lestgabo can we please capitalize "To: and "From:"?

Reply:
I wouldn't know how to capitalize correctly for other languages so I will only do so for the English version :)

Reply:
@esethna done